org.gnome.Chess.json

```diff
@@ -13,6 +13,45 @@
     ],
     "cleanup": ["/share/gnuchess", "/share/info", "/share/man", "/include"],
     "modules": [
+        {
+            "name": "stockfish",
+            "buildsystem": "simple",
+            "build-options" : {
+                "arch": {
+                    "x86_64": {
+                        "env": {
+                            "ARCH": "x86-64"
+                        }
+                    },
+                    "i386": {
+                        "env": {
+                            "ARCH": "x86-32"
+                        }
+                    },
+                    "arm": {
+                        "env": {
+                            "ARCH": "armv7"
+                        }
+                    },
+                    "aarch64": {
+                        "env": {
+                            "ARCH": "armv7"
```

Comment:
Looks like this worked; at least the aarch64 build succeeded. It's pretty evil though; would be better to patch the Makefile.

I wonder why Stockfish is using a custom Makefile instead of a proper build system. :/